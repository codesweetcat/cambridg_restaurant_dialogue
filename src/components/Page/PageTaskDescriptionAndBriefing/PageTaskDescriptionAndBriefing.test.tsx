import renderer from 'react-test-renderer';
import TaskDescritionAndBriefing from './TaskDescriptionAndBriefing'

test('PageTaskDescriptionAndBriefing renders correctly', () => {
  const tree = renderer
    .create(<TaskDescritionAndBriefing />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
