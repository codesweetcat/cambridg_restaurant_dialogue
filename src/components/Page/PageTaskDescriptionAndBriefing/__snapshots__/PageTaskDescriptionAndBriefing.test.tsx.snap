// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTaskDescriptionAndBriefing renders correctly 1`] = `
<div
  className="container searchContainer"
>
  <div>
    It looks like something went wrong.  Please reload this page to start over. Remember to try and get the system to recommend a restaurant that matches the task description.
  </div>
</div>
`;
